adit/dicom_web/tests/acceptance/test_wadors.py

```diff
@@ -37,6 +40,63 @@ def test_retrieve_study(channels_live_server: ChannelsLiveServer):
     )
 
 
+@pytest.mark.acceptance
+@pytest.mark.order("last")
+@pytest.mark.django_db(transaction=True)
+def test_retrieve_study_with_pseudonym(channels_live_server: ChannelsLiveServer):
+    setup_dimse_orthancs()
+
+    _, group, token = create_user_with_dicom_web_group_and_token()
+    server = DicomServer.objects.get(ae_title="ORTHANC1")
+    grant_access(group, server, source=True)
+    orthanc1_client = create_dicom_web_client(channels_live_server.url, server.ae_title, token)
```

Comment:
With adit_client you don't need this `create_dicom_web_client` anymore. You can just use `adit_client = AditClient(server_url=channels_live_server.url, auth_token=token)`. `server.ae_title` must be provided then to the retrieve method of adit_client.